src/data/chats/mod.rs

```diff
@@ -154,21 +160,14 @@ impl Chats {
     }
 
     pub fn create_empty_chat_and_load_file(&mut self, file: &File) {
-        let new_chat = RefCell::new(Chat::new(self.chats_dir.clone()));
-        new_chat.borrow().save();
-
-        self.cancel_chat_streaming();
+        let mut new_chat = Chat::new(self.chats_dir.clone());
+        new_chat.last_used_file_id = Some(file.id.clone());
```

Comment:
Technically, `load_model` does the work of this 2 lines, but I prefer being explicit here, and optimize `load_model` to avoid writing the file again if the last used model didn't change.